frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -162,6 +169,9 @@
         {/if}
 
         <ChangeMemberRoleModal projectId={project.id} bind:this={changeMemberRoleModal} />
+        {#if isAdmin(user)}
```

Comment:
These's a "pretty" component for that.
```suggestion
        <AdminContent>
```